src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -193,6 +193,58 @@ On the Settings page, you can change your business information. Please note this
 
 ![Centrapay business portal settings page](/centrapay-portal-settings.png)
 
+## FAQs
+
+- **What do we do if a customer returns one or more items, and the invoice has already been sent to Farmlands?**
+
+    Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.
+
+- **What do we do if the customer cancels their order after we have sent the invoice to Farmlands?**
+
+    Raise a credit note as normal and insert the original authorisation short code on the invoice. This will enable us to identify the original invoice and customer account details.
+
+- **What do we do if the customer cancels their order before we have sent the invoice to Farmlands?**
+
+    If the entire order has been cancelled, release the hold on the customer’s funds via the Portal. There is no requirement to send an invoice to Farmlands.
```

Comment:
Not a now thing but it would be nice for these to link to parts of the portal guide. E.g. "release the hold on the customer's funds" could link to a section showing how to do a release.

Reply:
I'll catch up with design - Reason I didn't is the ToC sidebar gets extremely long - might want a way to have an anchor tag without it showing up in ToC